subprojects/precondition-tester/src/test/groovy/org/gradle/test/precondition/RemotePreconditionProbingTest.groovy

```diff
@@ -18,7 +18,9 @@ package org.gradle.test.precondition
 
 
 import com.gradle.enterprise.testing.annotations.RemoteOnly
+import spock.lang.Ignore
 
 @RemoteOnly
+@Ignore("https://github.com/gradle/gradle-private/issues/3905")
```

Comment:
I'm afraid that we will forget about this issue forever.
Do we have a plan for how this can be fixed in the future?

Asking as this `@Ignore` impairs our observations severely, as we don't monitor remote agents anymore by doing this change.